src/server/scripts/Northrend/Naxxramas/boss_loatheb.cpp

```diff
@@ -177,49 +181,6 @@ class achievement_spore_loser : public AchievementCriteriaScript
         }
 };
 
-typedef boss_loatheb::boss_loathebAI LoathebAI;
-
-class spell_loatheb_necrotic_aura_warning : public SpellScriptLoader
-{
-    public:
-        spell_loatheb_necrotic_aura_warning() : SpellScriptLoader("spell_loatheb_necrotic_aura_warning") { }
-
-        class spell_loatheb_necrotic_aura_warning_AuraScript : public AuraScript
-        {
-            PrepareAuraScript(spell_loatheb_necrotic_aura_warning_AuraScript);
-
-            bool Validate(SpellInfo const* /*spell*/) override
-            {
-                if (!sSpellStore.LookupEntry(SPELL_WARN_NECROTIC_AURA))
-                    return false;
-                return true;
-            }
-
-            void HandleEffectApply(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
-            {
-                if (GetTarget()->IsAIEnabled)
-                    ENSURE_AI(LoathebAI, GetTarget()->GetAI())->Talk(SAY_NECROTIC_AURA_APPLIED);
-            }
-
-            void HandleEffectRemove(AuraEffect const* /*aurEff*/, AuraEffectHandleModes /*mode*/)
-            {
-                if (GetTarget()->IsAIEnabled)
-                    ENSURE_AI(LoathebAI, GetTarget()->GetAI())->Talk(SAY_NECROTIC_AURA_REMOVED);
-            }
-
-            void Register() override
-            {
-                AfterEffectApply += AuraEffectApplyFn(spell_loatheb_necrotic_aura_warning_AuraScript::HandleEffectApply, EFFECT_0, SPELL_AURA_DUMMY, AURA_EFFECT_HANDLE_REAL);
-                AfterEffectRemove += AuraEffectRemoveFn(spell_loatheb_necrotic_aura_warning_AuraScript::HandleEffectRemove, EFFECT_0, SPELL_AURA_DUMMY, AURA_EFFECT_HANDLE_REAL);
-            }
-        };
-
-        AuraScript* GetAuraScript() const override
-        {
-            return new spell_loatheb_necrotic_aura_warning_AuraScript();
-        }
-};
```

Comment:
in this example nothing, that happens only if the boss has a spell with casttime

since the emote depends on the aura, it should be controlled by this aura, it's my oppinion